<Xcpt>
  <Time>20.04.2017 08:08:42</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>UserDoesNotExist</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\SandManu\Documents\GitHub\LAP_Sandholzer\CardGame.DAL\Logic\AuthManager.cs:Zeile 56.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>20.04.2017 08:09:14</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>UserDoesNotExist</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\SandManu\Documents\GitHub\LAP_Sandholzer\CardGame.DAL\Logic\AuthManager.cs:Zeile 56.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>20.04.2017 08:10:28</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>UserDoesNotExist</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\SandManu\Documents\GitHub\LAP_Sandholzer\CardGame.DAL\Logic\AuthManager.cs:Zeile 56.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>20.04.2017 08:10:34</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>UserDoesNotExist</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\SandManu\Documents\GitHub\LAP_Sandholzer\CardGame.DAL\Logic\AuthManager.cs:Zeile 56.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>20.04.2017 13:56:21</Time>
  <Exception>
    <Source>EntityFramework</Source>
    <Message>In der Anwendungskonfigurationsdatei wurde keine Verbindungszeichenfolge mit dem Namen 'CardGame_v2Entities' gefunden.</Message>
    <Stack>   bei System.Data.Entity.Internal.LazyInternalConnection.get_ConnectionHasModel()
   bei System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   bei System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   bei System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   bei System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   bei System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   bei System.Linq.Queryable.Where[TSource](IQueryable`1 source, Expression`1 predicate)
   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\SandManu\Documents\GitHub\LAP_Sandholzer\CardGame.DAL\Logic\AuthManager.cs:Zeile 53.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>20.04.2017 13:57:37</Time>
  <Exception>
    <Source>EntityFramework</Source>
    <Message>In der Anwendungskonfigurationsdatei wurde keine Verbindungszeichenfolge mit dem Namen 'CardGame_v2Entities' gefunden.</Message>
    <Stack>   bei System.Data.Entity.Internal.LazyInternalConnection.get_ConnectionHasModel()
   bei System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   bei System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   bei System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   bei System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   bei System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   bei System.Linq.Queryable.Where[TSource](IQueryable`1 source, Expression`1 predicate)
   bei CardGame.DAL.Logic.AuthManager.AuthUser(String email, String password) in C:\Users\SandManu\Documents\GitHub\LAP_Sandholzer\CardGame.DAL\Logic\AuthManager.cs:Zeile 53.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>20.04.2017 14:01:14</Time>
  <Exception>
    <Source>EntityFramework</Source>
    <Message>In der Anwendungskonfigurationsdatei wurde keine Verbindungszeichenfolge mit dem Namen 'CardGame_v2Entities' gefunden.</Message>
    <Stack>   bei System.Data.Entity.Internal.LazyInternalConnection.get_ConnectionHasModel()
   bei System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   bei System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   bei System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   bei System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   bei System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   bei System.Linq.Queryable.Any[TSource](IQueryable`1 source, Expression`1 predicate)
   bei CardGame.DAL.Logic.AuthManager.Register(tblUser regUser) in C:\Users\SandManu\Documents\GitHub\LAP_Sandholzer\CardGame.DAL\Logic\AuthManager.cs:Zeile 19.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>20.04.2017 14:05:42</Time>
  <Exception>
    <Source>EntityFramework</Source>
    <Message>In der Anwendungskonfigurationsdatei wurde keine Verbindungszeichenfolge mit dem Namen 'CardGame_v2Entities' gefunden.</Message>
    <Stack>   bei System.Data.Entity.Internal.LazyInternalConnection.get_ConnectionHasModel()
   bei System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   bei System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   bei System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   bei System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   bei System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   bei System.Linq.Queryable.Any[TSource](IQueryable`1 source, Expression`1 predicate)
   bei CardGame.DAL.Logic.AuthManager.Register(tblUser regUser) in C:\Users\SandManu\Documents\GitHub\LAP_Sandholzer\CardGame.DAL\Logic\AuthManager.cs:Zeile 19.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>20.04.2017 14:14:15</Time>
  <Exception>
    <Source>EntityFramework</Source>
    <Message>In der Anwendungskonfigurationsdatei wurde keine Verbindungszeichenfolge mit dem Namen 'CardGame_v2Entities' gefunden.</Message>
    <Stack>   bei System.Data.Entity.Internal.LazyInternalConnection.get_ConnectionHasModel()
   bei System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   bei System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   bei System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   bei System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   bei System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   bei System.Linq.Queryable.Any[TSource](IQueryable`1 source, Expression`1 predicate)
   bei CardGame.DAL.Logic.AuthManager.Register(tblUser regUser) in C:\Users\SandManu\Documents\GitHub\LAP_Sandholzer\CardGame.DAL\Logic\AuthManager.cs:Zeile 19.</Stack>
  </Exception>
</Xcpt>
